components/src/utils/datePickerLocales.js

```diff
@@ -0,0 +1,7 @@
+import { registerLocale } from "react-datepicker";
+import { enUS, de, fr, es, ptBR, da, pl } from "date-fns/locale";
+
+export const supportedDateLocales = { enUS, de, fr, es, ptBR, da, pl };
```

Comment:
There are fewer languages then we actually had in the list since date-fns doesn't make a distinction between french (fr) and spanish (es) in different countries.